src/components/ProcessCreate/Steps/Confirm.tsx

```diff
@@ -243,19 +242,14 @@ export const Confirm = () => {
         </Button>
         <Modal isOpen={isOpen} onClose={onClose} closeOnEsc={!!error} closeOnOverlayClick={!!error} isCentered>
           <ModalOverlay />
-          <ModalContent>
-            <ModalHeader>{t('form.process_create.creating_process')}</ModalHeader>
+          <ModalContent p={12}>
+            <ModalHeader p={0} mb={5} textAlign='center'>
+              {t('form.process_create.creating_process')}
+            </ModalHeader>
             {error && <ModalCloseButton />}
-            <ModalBody>
+            <ModalBody p={0}>
```

Comment:
Shouldn't these modal styles be defined in the theme for all other modals?

Reply:
Opened a new issue to track this: https://github.com/vocdoni/ui-scaffold/issues/323